torchao/quantization/quantize_/workflows/float8/float8_tensor.py

```diff
@@ -617,6 +617,22 @@ def _(func, types, args, kwargs):
     return return_and_correct_aliasing(func, args, kwargs, new)
 
 
+@implements(aten.select.int)
+def _(func, types, args, kwargs):
+    old_float8_tensor, dim, index = args
+    assert dim == 0, f"Float8Tensor aten.select.int with {dim=} is not yet supported"
+    new_float8_tensor = old_float8_tensor.__class__(
```

Comment:
nit: is it better to use `Float8Tensor` directly here? or `old_float8_tensor.__class__`? we have been using `Float8Tensor` in other functions so far